Add react-route-dom to switch between pages

victorleung1119 opened the pull request:
Add react-route-dom to switch between 'home', 'about', and 'blog'.
Add additional CSS to Header.css to prevent the <Link> from altering the Header.js css.

The link was not added to the 'job' button because it is supposed to link to a specific section in the 'about' page. The about page needs to be built first before it can reference to that part of the 'about' page.

victorleung1119 committed:
Add react-route-dom to switch between pages

Add react-route-dom to switch between 'home', 'about', and 'blog'.
Add additional CSS to Header.css to prevent the <Link> from altering the Header.js css.

The link was not added to the 'job' button because it is supposed to link to a specific section in the 'about' page. The about page needs to be built first before it can reference to that part of the 'about' page.

victorleung1119 committed:
Change the header to be sticky

Change the header to be sticky.
Still need to customize the color of the header to be #fafafa

Blazekla commented:
Routing works! Great job!

Could you get the mobile hamburger icon to work similar to the desktop links?
The hamburger icon currently shows on both desktop and mobile screen sizes and has not functionality.

victorleung1119 committed:
Make the mobile links functional

1. Make the mobile links functional.
2. Create function and state to change active button class on different pages.
3. Delete the extra burger menu button. Make the only burger button 'display: none' at large viewport

victorleung1119 committed:
Correct a bug in the header

Correct a bug in the header
The initial state for the activeButton is now 'Home' instead of 'About'
